source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -455,15 +456,36 @@ For more detailed information, visit our :ref:`telemetry-about` page.
 
       Figure 14: Enable Telemetry
 
+Recommended options
+===================
+
+After you complete the `Required options`_, we highly recommend completing a
+few `Advanced Options`_ to improve your desktop or developer experience.
+
+Select:
+
+* `Bundle Selection`_ to add :file:`desktop-autostart`, which launches the
```

Comment:
@mvincerx , you mention here to add the desktop-autostart bundle but if you go to the section below (Bundle selection) you only show the "R-extras" bundle as selected.  You should reiterate in that section that we highly recommend they add the desktop-autostart bundle (and I would argue the user-basic bundle since it gets the bulk of useful bundles without adding os-clr-on-clr).  I think this would present a more consistent message